sklearn/linear_model/coordinate_descent.py

```diff
@@ -1537,15 +1556,17 @@ def __init__(self, l1_ratio=0.5, eps=1e-3, n_alphas=100, alphas=None,
 class MultiTaskElasticNet(Lasso):
     """Multi-task ElasticNet model trained with L1/L2 mixed-norm as regularizer
 
-    The optimization objective for MultiTaskElasticNet is::
+    The optimization objective for MultiTaskElasticNet is
 
-        (1 / (2 * n_samples)) * ||Y - XW||^Fro_2
-        + alpha * l1_ratio * ||W||_21
-        + 0.5 * alpha * (1 - l1_ratio) * ||W||_Fro^2
+    .. math::
+        \\frac{1}{2 * \\mathrm{n\\_samples}} * \\|Y - XW\\|^2_{\\mathrm{Fro}}
+        + \\alpha * \\mathrm{l1\\_ratio} * \\|W\\|_{2,1}
+        + 0.5 * \\alpha * (1 - \\mathrm{l1\\_ratio}) * \\|W\\|_{\\mathrm{Fro}}^2
```

Comment:
This is the line that goes over 79 characters and makes Travis go red. It's worth invest some time to configure on-the-fly flake8 checks inside your editor of choice. This allows you to spot these kind of problems while editing.